en/_posts/2023-09-29-groonga-13.0.8.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post.en
+title: Groonga 13.0.8 has been released
+description: Groonga 13.0.8 has been released!
+published: false
+---
+
+## Groonga 13.0.8 has been released
+
+[Groonga 13.0.8](/docs/news.html#release-13-0-8) has been released!
+
+How to install: [Install](/docs/install.html)
+
+### Changes
+
+Here are important changes in this release:
+
+### Improvements
+
+* [[column_create](/docs/reference/commands/column_create.html)] Improved `column_create` flags with addition new flags `COLUMN_FILTER_SHUFFLE`, `COLUMN_FILTER_BYTE_DELTA`, `COMPRESS_FILTER_TRUNCATE_PRECISION_1BYTE`, and `COMPRESS_FILTER_TRUNCATE_PRECISION_2BYTES`.
+
```

Comment:
```suggestion

These new flags are for compression rate of `COMPRESS_ZBLIB`, `COMPRESS_ZSTD` to increase by filtering the data before the compression. Even though applying these flags, the compression rate may not increase depends on the data . Also, applying these  flags on the filters causes delay to  save and reference the column because the using filters with these flags causes additional process.  

It is important to choose and use effective filters only.

Regardless of setting `COMPRESS_ZLIB`, `COMPRESS_LZ4`,` COMPRESS_ZSTD`, these filters use [BloscLZ](https://www.blosc.org/pages/blosc-in-depth/#blosc-as-a-meta-compressor) as the compression alogolizm, thus  make the column size smaller than the noncompressed column in most cases. Yet, there would be a case that some data would show sufficient work by only suetting `COMPRESS_ZLIB`, `COMPRESS_LZ4`,` COMPRESS_ZSTD` .  So it is advised to set suitable flags depending on the data.

These flags are only available with `COLUMN_VECTOR` and are ignored with `COLUMN_SCALAR`.

Here are a list of  which kind of data is suitable for each filter.

  * `COLUMN_FILTER_SHUFFLE`
    This filter focus on the N-th byte element to put in order.
    This filter places data in continuance gathered from only 0th byte of each element within the vector column, then from only 1st byte, and then from only 2nd byte. It repeats gathering and placing the data in continuance for every bytes.

    The point for this filter is reorganized data in continuance with same value because data are gathered based on each N byte. It is because compressed algorithm of `COMPRESS_ZLIB`, `COMPRESS_LZ4`,and ` COMPRESS_ZSTD` tend to increase compression rate for the continuance data with same value.
    
   Here is how to handle in example.    
```

Reply:
"
```

```
"
が使えなかったので、表（？）がそのままになっています。すみません。
インデントもうまくいってなかったら申し訳ありません。

Reply:
ありがとうございます。表とインデントはいい感じにしておくので気にしなくて大丈夫です。

>英語は同じ主語の同じ構文を連続するのを好まないので「このフラグは、圧縮対象の値のバイト間の差分を計算するフラグです。 このフラグは、隣り合うバイトの差分を計算します。」を主語一個で訳してしまいましたが、もし前の文を行うために、具体的には次の分を行う。という意図である場合は文章を変えるので教えてください。　

ここは、「前の文を行うために、具体的には次の分を行う。」という意図なので、文章を変えていただけると助かります。